### PR TITLE
README.md: Update url from coala-analyser to coala

### PR DESCRIPTION
**Pull request by kaustubhhiware**
README.md: Update url from coala-analyser to coala

Correct the broken link in README

Closes https://github.com/coala/docker-coala-base/issues/92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,5 +12,5 @@ like GitLab CI and Travis CI are available in
 The cache generated by coala are placed on `/cache`. You can preserve it by
 creating a volume for it, like this:
 
-`docker run -v=$(pwd):/app -v=/tmp/coala:/cache --workdir=/app coala/base 
+`docker run -v=$(pwd):/app -v=/tmp/coala:/cache --workdir=/app coala/base
```
Comment (Udayan12167) [inline README.md]: This is an unrelated change to this commit. Make a separate commit to remove the trailing whitespace please?

**Comment by kaustubhhiware** [comment]
Ping @Udayan12167 , I have restored the whitespace. Could you please review this PR ?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 coala
 =====
 
-This image has coala (coala-analyzer.org) and all official bears and
+This image has [coala](http://coala.io/) (coala.io) and all official bears and
```
Comment (satwikkansal) [inline README.md]: No need to specify "(coala.io)" additionally :)

**Comment by jayvdb** [comment]
Please fix and rebase.

**Comment by kaustubhhiware** [comment]
Is it ok if I made another PR for this ? 
I realised my mistake @jayvdb
